src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +15,26 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+    private final String identifier = "w";
```

Comment:
please remember about correct constant field namings
```suggestion
    private static final String IDENTIFIER = "w";
```